tests/test_other.py

```diff
@@ -9780,6 +9780,34 @@ def test_assertions_on_outgoing_module_api_changes(self):
 Module.arguments has been replaced with plain arguments_
 ''', run_js('a.out.js', assert_returncode=None, stderr=PIPE))
 
+  def test_em_asm_duplicate_strings(self):
+    # We had a regression where tow different EM_ASM strings from two diffferent
```

Comment:
Typo: s/tow/two/

Should we link to the issue here? Should add even more context

Reply:
Err, these is no issue :(